<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreStartGame</class>
 <widget class="QDialog" name="PreStartGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="RadioDifficile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Difficile</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RadioFacile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Facile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartGame">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>560</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Lancer la partie</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RadioNormal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </widget>
  <widget class="QListView" name="ListMusic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>471</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditChemin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>471</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditProgress">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>560</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>100</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
